Gemfile

```
@@ -5,12 +5,9 @@ gemspec
 gem 'danger', '~> 8.0'
 gem 'danger-rubocop', '~> 0.6'
 
-group :test do
-  gem 'codecov', require: false
-  gem 'rspec'
-  gem 'webmock', require: false, group: :test
-  gem 'yard'
-end
+gem 'codecov', require: false
+gem 'webmock', require: false, group: :test
```

```suggestion
gem 'webmock', require: false
```
Not sure why the group was repeated in the original code, likely an oversight.